packages/lesswrong/components/tagging/subforums/SubforumWikiTab.tsx

```diff
@@ -135,18 +126,11 @@ const SubforumWikiTab = ({tag, revision, truncated, setTruncated, classes}: {
         {tag.sequence && <TagIntroSequence tag={tag} />}
         {!tag.wikiOnly && (
           <AnalyticsContext pageSectionContext="tagsSection">
-            {tag.sequence ? (
-              <SectionTitle title={`Posts tagged ${tag.name}`}>
-                <PostsListSortDropdown value={query.sortedBy || "relevance"} />
-              </SectionTitle>
-            ) : (
-              <div className={classes.tagHeader}>
-                <div className={classes.postsTaggedTitle}>
-                  Posts tagged <em>{tag.name}</em>
-                </div>
-                <PostsListSortDropdown value={query.sortedBy || "relevance"} />
-              </div>
-            )}
+            <SectionTitle title={`Posts tagged ${tag.name}`} />
+            <div className={classes.postListMeta}>
+              <PostsListSortDropdown value={query.sortedBy || "relevance"} />
+              <div className={classes.relevance}>Relevance</div>
+            </div>
```

Comment:
Ew, are we really still copy pasting this file? Sigh.

Reply:
Yeah, honestly I have no idea what's going on with the structure here - I think it's refelecting a page taxonomy that no longer exists(?) which is kind of confusing.